deployment/clouddeploy/deployer.go

```diff
@@ -543,6 +822,98 @@ func (d *Deployer) UpdateClusterExpiry(ctx context.Context, clusterID string, ne
 	return nil
 }
 
+func CompareUpdateClusterSpecs(spec1, spec2 capellacontrol.UpdateClusterSpecsRequest_Spec) int {
```

Comment:
nit: We should make these helper functions private (`compareUpdateClusterSpecs`), or move them to a separate file to declutter this file (it's gotten quite large and difficult to read through).

Reply:
okay, i will create a file inside clouddeploy named clouddeploy-helper.go and move this functions to that

Reply:
It's already in the clouddeploy folder, so you could just call it `helpers.go`.

Reply:
yeah